app/views/users/index.html.erb

```diff
@@ -1 +1,21 @@
-<h1>List of User "Index Page"</h1>
\ No newline at end of file
+<h1>List of User "Index Page"</h1>
+<table style="width: 100%">
+  <thead>
+    <tr>
+      <td><strong>Image</strong></td>
+      <td><strong>Name</strong></td>
+      <td><strong>Number of Posts</strong></td>
+    </tr>
+  </thead>
+  <tbody>
+    <% @users.each do |user| %>
+    <tr>
+      <td>
+        <img src="#{<%= user.photo %>}" alt="User Img" />
+      </td>
+      <td><%= user.name %></td>
+      <td><%= user.posts_counter %></td>
+    </tr>
```

Comment:
- [ ] Kindly add a link for each user when you click on it suppose takes you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png)

Reply:
Link is added 👍🏻